service/src/main/java/bio/terra/profile/service/profile/ProfileService.java

```diff
@@ -233,7 +238,15 @@ private Organization getProfileOrganization(BillingProfile profile) {
             profile
                 .subscriptionId()
                 .map(enterpriseConfiguration.subscriptions()::contains)
-                .orElse(false));
+                .orElse(false))
+        .limits(
+            profile
+                .subscriptionId()
+                .flatMap(
+                    id ->
+                        Optional.ofNullable(limitsConfiguration.subscriptions())
+                            .map(subscriptions -> subscriptions.get(id)))
+                .orElse(Map.of()));
```

Comment:
Could this be moved into `LimitsConfiguration` and exposed via a `getLimitsForSubscription` method or something like that?

Reply:
Spring is yelling at me that a `ConfigurationProperties` has to be a record, which means I can't add additional methods to it.  Am struggling with making config that's not a record, it might be possible!